ext/session/tests/session_set_save_handler_iface_002.phpt

```diff
@@ -86,5 +86,5 @@ session_start();
 bool(true)
 
 Warning: session_set_save_handler() expects parameter 1 to be SessionHandlerInterface, object given in %s
-bool(false)
```

Comment:
Why should the feature for user defined serialized affect the return value of this method?